From bb8ca82b4c94f768a082a5832bb6573300050eff Mon Sep 17 00:00:00 2001
From: Fang Wang <fang.c.wang@intel.com>
Date: Wed, 20 May 2015 10:39:26 +0800
Subject: [PATCH 399/462] OV8858: Remove the 1080P@30fps for preview

On ov8858, 1080P sensor setting dosen't output full resolution,
so remove it.

For CTS verfier FOV calibration case, when running offline mode,
driver will select the sensor setting in preview.

Change-Id: Ib4182d9a50cfa54468a269215fc59dec2fb14ef7
Tracked-On: https://jira01.devtools.intel.com/browse/IMINAN-34394
Signed-off-by: Fang Wang <fang.c.wang@intel.com>
Reviewed-on: https://icggerrit.ir.intel.com/56719
Approver: Jian Xu Zheng <jian.xu.zheng@intel.com>
Reviewed-by: Jian Xu Zheng <jian.xu.zheng@intel.com>
Integrator: WeiX Meng <weix.meng@intel.com>
Build: IMIN (SC)
Reviewed-by: Soren Friis <soren.friis@intel.com>
Build: WeiX Meng <weix.meng@intel.com>
Reviewed-by: WeiX Meng <weix.meng@intel.com>
Maintainer: WeiX Meng <weix.meng@intel.com>
---
 .../camera/drivers/media/i2c/ov8858.h                 | 19 -------------------
 1 file changed, 19 deletions(-)

diff --git a/drivers/external_drivers/camera/drivers/media/i2c/ov8858.h b/drivers/external_drivers/camera/drivers/media/i2c/ov8858.h
index 0194ceb..2af5928 100644
--- a/drivers/external_drivers/camera/drivers/media/i2c/ov8858.h
+++ b/drivers/external_drivers/camera/drivers/media/i2c/ov8858.h
@@ -843,25 +843,6 @@ static const struct ov8858_reg ov8858_1080P_30[] = {
 
 static struct ov8858_resolution ov8858_res_preview[] = {
 	{
-		.desc = "ov8858_1080P_30_PREVIEW",
-		.width = 1936,
-		.height = 1096,
-		.used = 0,
-		.regs = ov8858_1080P_30,
-		.bin_factor_x = 0,
-		.bin_factor_y = 0,
-		.skip_frames = 1,
-		.fps_options =  {
-			{
-				.fps = 30,
-				.pixels_per_line = 3880,
-				.lines_per_frame = 2573,
-			},
-			{
-			}
-		},
-	},
-	{
 		.desc = "ov8858_6M_PREVIEW",
 		.width = 3280,
 		.height = 1852,
-- 
1.9.1

